test/cql-pytest/test_tablets.py

```diff
@@ -174,3 +174,38 @@ def test_tablets_are_dropped_when_dropping_view(cql, test_keyspace, skip_without
 # Test that when a tablets-enabled table drop fails when it has a view depending on it, all of its tablets still exist after the error is returned.
 def test_tablets_are_not_dropped_when_dropping_table_with_view_fails(cql, test_keyspace, skip_without_tablets):
     _test_tablets_are_dropped_when_dropping_table_with_view(cql, test_keyspace, attempt_drop_table=True)
+
+
+def _test_tablets_are_dropped_when_dropping_index(cql, keyspace_name:str, drop_index:bool):
+    table_name = unique_name()
+    schema = "pk int PRIMARY KEY, c int"
+    cql.execute(f"CREATE TABLE {keyspace_name}.{table_name} ({schema})")
+    index_name = unique_name()
+    cql.execute(f"CREATE INDEX {index_name} ON {keyspace_name}.{table_name} (c)")
```

Comment:
Nitpick: I'm starting to be sorry I asked you not to create a keyspace. If your test uses a common keyspace, it should be careful not to leave behind tables etc. in this keyspace when it fails. And this test isn't careful about that.

But I don't insist. We don't expect the test to fail anyway, and it's not like the DynamoDB tests where we actually pay with real money for every table that we leave behind :-)

Reply:
It's not very important. In Alternator it's very important (leaving behind test tables on AWS DynamoDB costs us money) but in CQL, if you don't leave many of them behind (and you won't, unless many tests fail), it's not a big problem.